lib/_stream_transform.js

```diff
@@ -124,7 +124,7 @@ function prefinish() {
   }
 }
 
-Transform.prototype._transform = function(chunk, encoding, callback) {
```

Comment:
Nit: Should there add the same comment `Abstract method. ....`?

Reply:
btw, keep the unused params as usual here could more clearly that imply how this abstract function be called. I think

Reply:
@Himself65 Seems like, we need to restore this change. So will do that.

Reply:
@mcollina Do I need to even revert this? Does these params gives the caller an understanding of how it should be called?

Reply:
Exactly, thanks

Reply:
@mcollina reverted both.